tests/unit/OmiseTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+define('_PS_VERSION_', 'TEST_VERSION');
+
+class OmiseTest extends PHPUnit_Framework_TestCase
+{
+    private $omise;
+
+    public function setup()
+    {
+        $paymentModule = $this->getMockBuilder(stdClass::class)
+            ->setMockClassName('PaymentModule')
+            ->setMethods(array('__construct', 'l'))
+            ->getMock();
+
+        $this->omise = new Omise();
+    }
+
+    public function testName_omise()
```

Comment:
@nimid for the naming of tests, it's better to explicit your test behaviour by have `test scenario` & `expectation` in the test name, and use english sentence pattern there, like
`testThatTheOmiseNamePropertyShouldReturnOmise`

Btw, some pattern says, you should have `test scenario` at the name, some pattern says you should have both `test scenario` & `expectation` at the name (and some says, use english sentence like ,"should", "must", etc in the test) to make it clear and it give us a benefit that we can easily understand the test by reading from the test name when it fails.

For me, I'm fine both, 
just it should be understandable when we skim through the lines without check the code to understand what's the test aim to prove.

Btw, I usually do like
```php
/**
 * @test
 */
public function people_should_happy_when_they_get_dog
{
    // assertion.
}

// or

/**
 * @test
 */
public function someone_must_feed_the_dog_food_when_they_bark
{
    // assertion.
}
```

I usually move `test` prefix to `@ test` annotation and use `_` to concat the string to make it easy to read and looks like english sentence instead of name it with `testCamelCase`.